src/pages/Account/Admin.tsx

```diff
@@ -73,49 +77,84 @@ const VenueDetails: React.FC<VenueDetailsProps> = ({ venueId }) => {
 
   return (
     <>
-      <ul>
+      <div className="page-container-adminpanel-tabs">
         {[
           { url: `${match.url}`, label: "Venue Infos" },
-          { url: `${match.url}/appearance`, label: "Appearance" },
+          // { url: `${match.url}/appearance`, label: "Appearance" },
           { url: `${match.url}/events`, label: "Events" },
         ].map((tab) => (
-          <li
+          <div
             key={tab.url}
-            style={location.pathname === tab.url ? { color: "red" } : {}}
+            className={`page-container-adminpanel-tab ${
+              location.pathname === tab.url ? "selected" : ""
+            }`}
           >
             <Link to={tab.url}>{tab.label}</Link>
-          </li>
+          </div>
         ))}
-      </ul>
-      <Switch>
-        <Route
-          path={`${match.url}/events`}
-          render={() => <EventsComponent venue={venue} />}
-          venue={venue}
-        />
-        <Route
-          path={`${match.url}/Appearance`}
-          component={() => <>Appearance Component</>}
-        />
-        <Route
-          path={`${match.url}`}
-          component={() => <VenueInfosComponent venue={venue} />}
-        />
-      </Switch>
+      </div>
+      <div className="page-container-adminpanel-venuepage">
+        <Switch>
+          <Route
+            path={`${match.url}/events`}
+            render={() => <EventsComponent venue={venue} />}
+            venue={venue}
+          />
+          <Route
+            path={`${match.url}/Appearance`}
+            component={() => <>Appearance Component</>}
+          />
+          <Route
+            path={`${match.url}`}
+            component={() => <VenueInfosComponent venue={venue} />}
+          />
+        </Switch>
+      </div>
     </>
   );
 };
 
 const VenueInfosComponent: React.FC<VenueDetailsPartProps> = ({ venue }) => {
   return (
     <>
-      <Link
-        to={`/v/${createUrlSafeName(venue.name)}`}
-        target="_blank"
-        rel="noopener noreferer"
-      >
-        Preview
-      </Link>
+      <div className="page-container-adminpanel-content">
+        <div className="container venue-entrance-experience-container">
+          <div
+            className="header"
+            style={{
+              background: `linear-gradient(
+            0deg,
+            rgba(0, 0, 0, 0.8) 2%,
+            rgba(0, 0, 0, 0) 98%
```

Comment:
prefer if this was defined in the scss file

Reply:
Don't think we can, it uses the image from from the venue config (just below)